float64 roll_angle
